tests/test_metadata_cache.py

```diff
@@ -16,6 +16,7 @@
 from __future__ import print_function
 
 import unittest
+from freezegun import freeze_time
```

Comment:
Probably, but the examples I have seen so far are way more complicated than freezegun